### PR TITLE
Update v2.6 missing Chinese translations

### PR DESCRIPTION
**Pull request by STARRY-S**
Update missing Chinese translation for v2.6:
1. `reference-guides/best-practices/best-practices.md`
1. `reference-guides/best-practices/rancher-server/rancher-server.md`
1. `reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md`
1. `reference-guides/best-practices/rancher-managed-clusters/rancher-managed-clusters.md`
1. `reference-guides/rancher-manager-architecture/rancher-manager-architecture.md`
1. `reference-guides/cluster-configuration/cluster-configuration.md`
1. `reference-guides/cluster-configuration/rancher-server-configuration/rancher-server-configuration.md`
1. `reference-guides/cluster-configuration/rancher-server-configuration/gke-cluster-configuration/gke-cluster-configuration.md`
1. `reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md`
1. `reference-guides/cluster-configuration/downstream-cluster-configuration/downstream-cluster-configuration.md`
1. `reference-guides/cluster-configuration/downstream-cluster-configuration/node-template-configuration/node-template-configuration.md`
1. `reference-guides/cluster-configuration/downstream-cluster-configuration/machine-configuration/machine-configuration.md`
1. `reference-guides/single-node-rancher-in-docker/single-node-rancher-in-docker.md`
1. `reference-guides/backup-restore-configuration/backup-restore-configuration.md`

@JacieChao Please help to review the docs when you have free time. Thanks!

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/backup-restore-configuration/backup-restore-configuration.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Rancher 备份配置参考
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/reference-guides/backup-restore-configuration"/>
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/backup-restore-configuration/backup-restore-configuration.md:6]: missing /zh

**Review by JacieChao — COMMENTED** [review]
每个文件的url都缺失了 `/zh`

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/backup-restore-configuration/backup-restore-configuration.md (hunk 1 above)
Comment (STARRY-S) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.6/reference-guides/backup-restore-configuration/backup-restore-configuration.md]: Done

**Review by JacieChao — COMMENTED** [review]
LGTM

**Comment by JacieChao** [comment]
Hi, @btat @martyav 

Thanks to @dingluyy @ly5156 @jianghang8421 @rootwuj @smallteeths @GGGitBoy @STARRY-S for their translation work for v2.6.

The relevant PRs are ready for merge:

- https://github.com/rancher/rancher-docs/pull/1276
- https://github.com/rancher/rancher-docs/pull/1275
- https://github.com/rancher/rancher-docs/pull/1274
- https://github.com/rancher/rancher-docs/pull/1264
- https://github.com/rancher/rancher-docs/pull/1263
- https://github.com/rancher/rancher-docs/pull/1256
- https://github.com/rancher/rancher-docs/pull/1255
- https://github.com/rancher/rancher-docs/pull/1262

cc @Jason-ZW

**Review by btat — APPROVED** [review]

**Comment by btat** [comment]
> Hi, @btat @martyav
> 
> Thanks to @dingluyy @ly5156 @jianghang8421 @rootwuj @smallteeths @GGGitBoy @STARRY-S for their translation work for v2.6.
> 
> The relevant PRs are ready for merge:
> 
>     * [Update missing Chinese translations of v2.6 #1276](https://github.com/rancher/rancher-docs/pull/1276)
> 
>     * [Update v2.6 missing Chinese translations #1275](https://github.com/rancher/rancher-docs/pull/1275)
> 
>     * [update missing chinese translation for v2.6 #1274](https://github.com/rancher/rancher-docs/pull/1274)
> 
>     * [Update missing translation about cis、istio、monitoring and logging parts for v2.6 #1264](https://github.com/rancher/rancher-docs/pull/1264)
> 
>     * [Update missing Chinese translation for v2.6 getting-started part #1263](https://github.com/rancher/rancher-docs/pull/1263)
> 
>     * [Update missing Chinese translation 2.7 and 2.6 #1256](https://github.com/rancher/rancher-docs/pull/1256)
> 
>     * [update missing Chinese translation for v2.6 #1255](https://github.com/rancher/rancher-docs/pull/1255)
> 
>     * [update missing Chinese translation for v2.6 #1262](https://github.com/rancher/rancher-docs/pull/1262)
> 
> 
> cc @Jason-ZW

Thank you for all the updates everyone.

**Pull request merged**